Snippets/Gateway/Gateway_3.1/UpgradeGuide/3to3.1/UpgradeGuideFeatureUsage.cs

```diff
@@ -0,0 +1,13 @@
+﻿using NServiceBus;
+
+class UpgradeGuideUsage
+{
+    void EnableGatewayAfter(EndpointConfiguration endpointConfiguration)
+    {
+        #region 2to3EnableGatewayAfter
```

Comment:
this isn't needed anymore, is it?

Reply:
It is, that is part of another upgrade guide.

Reply:
Removed all 3.0 snippets from 3.1